packages/kit/src/core/build/index.js

```diff
@@ -39,10 +39,11 @@ export async function build(config, { cwd = process.cwd(), runtime = '@sveltejs/
 		cwd,
 		config,
 		build_dir,
-		base:
-			config.kit.paths.assets === '/.'
-				? `/${config.kit.appDir}/`
-				: `${config.kit.paths.assets}/${config.kit.appDir}/`,
+		// TODO this is so that Vite's preloading works. Unfortunately, it fails
+		// during `svelte-kit preview`, because we use a local asset path. If Vite
+		// used relative paths, I _think_ this could get fixed. Issue here:
+		// https://github.com/vitejs/vite/issues/2009
+		base: `${config.kit.paths.assets || config.kit.paths.base}/${config.kit.appDir}/`,
```

Comment:
I might not understand what `base` is, but I'm surprised that `assets` affects it

Reply:
if `assets` is supplied, it's an absolute URL (like `https://cdn.example.com/stuff`). If not, assets are served from the same place as the rest of the app, which is either a specified base path like `/app-goes-here` or the empty string

Reply:
I thought by `base` that this would be the base URL for the application. Should we call it `assets_base` or something along those lines?

Reply:
makes sense, updated